src/main/scala/it/pagopa/interop/partyprocess/model/UserRegistryUser.scala

```diff
@@ -1,17 +1,24 @@
 package it.pagopa.interop.partyprocess.model
 
-import it.pagopa.userreg.client.model.UserResource
+import it.pagopa.userreg.client.model.{UserResource}
 
 import java.util.UUID
 
-final case class UserRegistryUser(id: UUID, name: String, surname: String, taxCode: String)
+final case class UserRegistryUser(
+  id: UUID,
+  name: String,
+  surname: String,
+  taxCode: String,
+  email: Map[String, String] = Map()
+)
 
 object UserRegistryUser {
   def fromUserResource(resource: UserResource): UserRegistryUser =
     UserRegistryUser(
       id = resource.id,
       taxCode = resource.fiscalCode.orNull,
       name = resource.name.map(n => n.value).orNull,
```

Comment:
Strongly suggest avoiding `null`s. Use `Option` instead.
If necessary, prefer defaulting to `null` just at the end of your flow.